polaris/loader/load.py

```diff
@@ -94,3 +95,22 @@ def load_benchmark(path: str, verify_checksum: ChecksumStrategy = "verify_unless
         benchmark.verify_checksum()
 
     return benchmark
+
+
+def load_competition(slug: SlugCompatibleStringType):
```

Comment:
**Suggestion**
This is not correct. Unfortunately, Python typing can't do the string template types we have in TypeScript. So this needs to be a `str`.
```suggestion
def load_competition(artifactId: str):
```